proto/viam/app/v1/app.proto

```diff
@@ -975,3 +979,23 @@ message CreateKeyResponse {
   string key = 1;
   string id = 2;
 }
+
+message DeleteKeyRequest {
+  string key = 1;
+  string id = 2;
+}
+
+message DeleteKeyResponse {}
+
+message ApiKey {
+  string key = 1;
+  string id = 2;
+}
+
+message ListKeysRequest {
+  string org_id = 1;
```

Comment:
just noticed this while looking at your PR for other reasons. Is the reason you are passing this in for if we are trying to list keys from somewhere that is not the APP? 

the org_id should be passed in within the context object so if this is from APP you probably don't need to explicitly pass this in the request.

Reply:
ah actually we are trying to remove support for current org in the BE so we should still explicitly include it here

Reply:
ah okay wasn't aware of that -- good to know thanks!

Reply:
for my own learnings, is there anything else we include in the context automatically and where in the code does this happen?

Reply:
i think it's everything that populates the state object here, it's mostly authentication/authorization information. except `currentOrg` lol, which we are trying to deprecate because it is unreliable and is only relevant for frontend state
https://github.com/viamrobotics/app/blob/0c1b93bf302d9ef4d70b9dd4995da8162622c15e/auth/state.go#L21-L32

Reply:
got it! thank you!!